Test/gsapTests/MockClasses.h

```diff
@@ -160,12 +160,13 @@ class TestPredictor final : public Predictor {
 public:
     TestPredictor(const PrognosticsModel& m,
                   LoadEstimator& le,
-                  TrajectoryService& trajService,
+                  const TrajectoryService& trajService,
                   const ConfigMap& config)
         : Predictor(m, le, trajService, config) {}
 
-    Prediction predict(double, const std::vector<UData>&) override {
-        ProgEvent event(MessageId::TestEvent0, {UData()}, {UData()});
+    Prediction predict(double, const std::vector<UData>& state) override {
+		auto prediction = UData(( (double) state[1].get() + (double) state[0].get() )/2);
```

Comment:
Shouldn't need casts here at all. If the are needed for some reason, prefer C++ style cast (in this case static_cast<double>())